pillar.example

```diff
@@ -31,3 +33,24 @@ lighttpd:
       - status
       - userdir
       - usertrack
+
+  tofs:
+    # The files_switch key serves as a selector for alternative
+    # directories under the formula files directory. See TOFS pattern
+    # doc for more info.
+    # Note: Any value not evaluated by `config.get` will be used literally.
+    # This can be used to set custom paths, as many levels deep as required.
+    files_switch:
+      - id
+      - osfinger
+      - os
+      - os_family
+    # All aspects of path/file resolution are customisable using the options below.
+    # This is unnecessary in most cases; there are sensible defaults.
+    # path_prefix: template_alt
+    # dirs:
+    #   files: files_alt
```

Comment:
Something has been bothering me about this `files` and the `files` below.  I'd like to change the name of one of these across all of the new TOFS before they become too widely used.  I think the bottom one should be changed to `templates`.  Is it too late to do this across the formulas where we've started using this?